### PR TITLE
Add additional domains to list of known shorteners

### PR DESCRIPTION
**Pull request by scottmatthewman**

### CODE REVIEW

File: embiggen.gemspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name = 'embiggen'
-  s.version = '1.2.3'
+  s.version = '1.2.4'
```
Comment (mudge) [inline embiggen.gemspec]: Can you please update the `README` to reflect the new version?

Comment (scottmatthewman) [inline embiggen.gemspec]: Ah, yes. Done in 631b8ef

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 A Ruby library to expand shortened URLs.
 
-**Current version:** 1.2.3  
+**Current version:** 1.2.4
```
Comment (mudge) [inline README.md]: Can you restore the two trailing spaces needed here to insert a [Markdown line-break](https://daringfireball.net/projects/markdown/syntax#p)?

Comment (jbilbo) [inline README.md]: Or use badgefury? 
```
[![Gem Version](https://badge.fury.io/rb/embiggen.svg)](https://badge.fury.io/rb/embiggen)
```
[Gem Version](https://badge.fury.io/rb/embiggen)

Comment (scottmatthewman) [inline README.md]: Whitespace restored. Also, `{  "trim_trailing_white_space_on_save": false }` added to my local `Markdown.sublime-settings` for the future…

**Pull request merged**